addons/mrp/tests/test_traceability.py

```diff
@@ -306,3 +306,83 @@ def test_tracking_on_byproducts(self):
         self.assertEqual(byproduct_move_line_2_lot_1.consume_line_ids.filtered(lambda l: l.qty_done), raw_line_raw_1_lot_1 | raw_line_raw_2_lot_1)
         byproduct_move_line_2_lot_2 = finished_move_lines.filtered(lambda ml: ml.lot_id.name == 'Byproduct_2_lot_2')
         self.assertEqual(byproduct_move_line_2_lot_2.consume_line_ids, raw_line_raw_1_lot_2 | raw_line_raw_2_lot_2)
+
+    def test_tracking_repair_production(self):
```

Comment:
Should be at post install or it won't be test in runbot.

Reply:
@ryv-odoo your wish is my command, it has been done and the checks are green :smile: